src/decisionengine_modules/htcondor/sources/job_q.py

```diff
@@ -1,14 +1,22 @@
+import traceback
```

Comment:
This line should be below `from collections import defaultdict`, otherwise I believe that the `black` autoformatter will complain and fail this PR.

Reply:
If black places it up here, that's fine then.